### PR TITLE
"Rename the app package"

### LINKED ISSUE #32: Rename the app package.
**Describe the feature you'd like**
Currently, the app package is `com.example.TempleApp`. Rename it to `org.amfoss.TempleApp` and made any other changes required for this.

**Would you like to work on the issue?** (Yes/No)
No.

### PR DESCRIPTION
**Pull request by athiranair2000**
Fixed #32 

Changes: Renamed app package from 'com.example.TempleApp' to 'org.amfoss.TempleApp'.

### CODE REVIEW

File: TempleApp/build.gradle [hunk 1]
```diff
@@ -20,15 +20,15 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13-beta-3'
```
Comment (Chromicle) [inline TempleApp/build.gradle]: Don't change gradle files

**Comment by immadisairaj** [comment]
@athiranair2000 resolve conflicts and make sure the travis build passes.

File: TempleApp/src/main/java/org/amfoss/templeapp/Activities/DeleteData.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+        delete.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+
+                if (flag == 1) {
+                    id = "REG" + uid1ET.getText().toString();
+                } else {
+                    id = "DON" + uid1ET.getText().toString();
```
Comment (Chromicle) [inline TempleApp/src/main/java/org/amfoss/templeapp/Activities/DeleteData.java]: you are behind of so many commits ex: strings resources
please make sure that you are up to date with project

**Comment by Chromicle** [comment]
please resolve conflicts and you build is also failing please check that

**Comment by immadisairaj** [comment]
@athiranair2000 The old files are not deleted. Can you check it once?
And You can also change the package `TempleApp/TempleApp` to `TempleApp/app`

**Comment by Chromicle** [comment]
> What are these files for?
> `TempleApp/src/androidTest/java/org/amfoss/templeapp/ApplicationTest.java`
> `TempleApp/src/test/java/org/amfoss/templeapp/ExampleUnitTest.java`

These all are fine, As the entire package is changed so they are also changed
They already there before only

**Comment by immadisairaj** [comment]
> > What are these files for?
> > `TempleApp/src/androidTest/java/org/amfoss/templeapp/ApplicationTest.java`
> > `TempleApp/src/test/java/org/amfoss/templeapp/ExampleUnitTest.java`
> 
> They all are fine, As the entire package is changed so they are also changed
> They are already there

But the location is also changed right. Let us see after @athiranair2000 pushes the changed code.

**Comment by yashk2000** [comment]
@athiranair2000 any updates?

**Comment by immadisairaj** [comment]
Good work

**Pull request closed without merging**